### ISSUE TITLE
Update `compact.svg`

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/arclight#938]
Remove the following from `compact.svg`:
 - [ ] `id`
 - [ ] `data-name`
 - [ ] `<defs><style>`
 - [ ] `title`

Blacklight v7.2.0 gives us a better ability to style and provide useful accessibility features.

**Comment by gordonleacock** [comment]
Un-assigning myself since I have to do some local umich work.

When I made this change I ended up with this for the compact.svg code:

`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path class="cls-1" d="M0,0H24V24H0Z"/><path d="M3,15H21V13H3Zm0,4H21V17H3Zm0-8H21V9H3ZM3,5V7H21V5Z"/></svg>`

and this as the image:
<img width="91" alt="issue938compact_icon_image" src="https://user-images.githubusercontent.com/12927191/67033052-d6e33380-f0e2-11e9-9f72-c84be4138a44.png">

Which didn't seem correct.
